pom.xml

```diff
@@ -78,6 +78,14 @@
         </exclusion>
       </exclusions>
     </dependency>
+
+    <!-- Included for backward compatibility, should be removed once downstream plugins are updated to use okhttp3 -->
+    <dependency>
+      <groupId>com.squareup.okhttp</groupId>
+      <artifactId>okhttp-urlconnection</artifactId>
```

Comment:
Do you need the other okhttp 2.x dependencies as well, or are they transitive dependencies of this one?

Reply:
Looks like the main OkHttp library is a transitive dependency and is bundled as desired:

```
[INFO] Bundling direct dependency okhttp-urlconnection-2.7.5.jar
[WARNING] Bundling transitive dependency okhttp-2.7.5.jar (via okhttp-urlconnection)
[WARNING] Bundling transitive dependency commons-lang3-3.9.jar (via github-api)
```